packages/helm-charts/blockscout/templates/blockscout-migration.hook.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   labels:
     {{- include "celo.blockscout.labels" . | nindent 4 }}
     component: blockscout-migration
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
```

Comment:
this is pretty cool, as I understand it this line is all that's needed to create a helm hook?

How is completion defined? is it the exit status of the contained process or some other mechanism that decides if the job is completed / failed?